### PR TITLE
Support new extension AMP.push format

### PR DESCRIPTION
**Pull request by dvoytenko**
Partial for #3742.

### CODE REVIEW

**Comment by cramforce** [comment]
LGTM

File: src/runtime.js [hunk 1]
```diff
@@ -172,46 +172,57 @@ function adoptShared(global, callback) {
...
   global.AMP.toggleExperiment = toggleExperiment.bind(null, global);
 
+  /**
+   * Sets the function to forward tick events to.
+   * @param {function(string,?string=,number=)} fn
+   * @param {function()=} opt_flush
+   * @deprecated
+   * @export
+   */
+  global.AMP.setTickFunction = (fn, opt_flush) => {};
```
Comment (erwinmombay) [inline src/runtime.js]: we should be able to remove this now, as the viewer no longer uses this.

cc @ericfs

Comment (erwinmombay) [inline src/runtime.js]: we can defer that work to later PR. just wanted to mention it

Comment (dvoytenko) [inline src/runtime.js]: Feel free to file a bug and send a PR :)

Comment (ericfs) [inline src/runtime.js]: SGTM

**Pull request merged**